bundles/EcommerceFrameworkBundle/Tools/Installer.php

```diff
@@ -414,4 +411,14 @@ public function needsReloadAfterInstall()
     {
         return true;
     }
+
+    /**
+     * @return Schema
+     */
+    public function getSchema() : ?Schema {
```

Comment:
If this returns `null` the calls that use this getter will fail.

Reply:
Resolved. Schema is required for installation anyway.